packages/beacon-node/src/chain/archiveStore/archiveStore.ts

```diff
@@ -111,34 +106,4 @@ export class ArchiverStore {
       this.chain.forkChoice.getFinalizedCheckpoint()
     );
   }
-
-  private onFinalizedCheckpoint = async (finalized: CheckpointWithHex): Promise<void> => {
-    return this.jobQueue.push(finalized);
-  };
-
-  private processFinalizedCheckpoint = async (finalized: CheckpointWithHex): Promise<void> => {
-    try {
-      const finalizedEpoch = finalized.epoch;
-      this.logger.verbose("Start processing finalized checkpoint", {epoch: finalizedEpoch, rootHex: finalized.rootHex});
-      if (this.opts.pruneHistory) {
-        await pruneHistory(
-          this.chain.config,
-          this.db,
-          this.logger,
-          this.metrics,
-          finalizedEpoch,
-          this.chain.clock.currentEpoch
-        );
-      }
-
-      this.prevFinalized = finalized;
```

Comment:
This was used for something before but dont remember seeing it reused in any of the previous PRs.  Why did we have this before and is it safe to not need it anymore?

Reply:
This was unused previously, we set it but never used it. 

https://github.com/ChainSafe/lodestar/blob/d60218eecbb05824f4a1138b1daf76ef4a9a0262/packages/beacon-node/src/chain/archiver/archiver.ts#L25

<img width="769" alt="image" src="https://github.com/user-attachments/assets/344a066d-dc21-4530-8e80-8527576bd7f5" />